LEGO1/mxstring.h

```diff
@@ -16,6 +16,8 @@ class MxString : public MxCore
   void ToLowerCase();
   const MxString &operator=(MxString *);
 
+  inline const char *Data() const { return m_data; }
```

Comment:
`GetData` is probably preferable to `Data`, according to the 96 source style guide